backend/build.gradle

```diff
@@ -62,3 +62,7 @@ tasks.named('test') {
 	useJUnitPlatform()
 	finalizedBy jacocoTestReport
 }
+
+configurations.all {
```

Comment:
스프링 부트 실행 시 아래와 같은 warning이 뜨는 것을 제거하기 위함입니다.
```
Standard Commons Logging discovery in action with spring-jcl: please remove commons-logging.jar from classpath in order to avoid potential conflicts
```

스프링 부트는 Spring-JCL을 통해 로깅 추상화를 제공해서, commons-logging을 명시적으로 포함하지 않아도 다른 의존성을 통해 간접적으로 포함될 수 있습니다. 이때, 잠재적으로 클래스 패스에서 충돌이 발생할 수 있어서 commons-logging 라이브러리를 제거해주는 설정을 해줬습니다.
* commons-logging 라이브러리를 포함하면 다시 컴파일하지 않고도 다른 로깅 구현으로 변경할 수 있다고 합니다.

[참고](https://mycodegate.com/1028/)